src/main/java/org/sagebionetworks/web/client/widget/team/JoinTeamWidgetViewImpl.java

```diff
@@ -222,10 +188,7 @@ public void setPresenter(Presenter presenter) {
 	
 	@Override
 	public void showJoinWizard() {
-		FlowPanel body = new FlowPanel();
-		body.add(progressWidget.asWidget());
-        body.add(currentWizardContent);
-		joinWizard.configure("Join", body, "Continue", DisplayConstants.BUTTON_CANCEL, new Dialog.Callback() {
+		joinWizard.configure("Join", "Continue", DisplayConstants.BUTTON_CANCEL, new Dialog.Callback() {
```

Comment:
The joinWizard was changed to be a 
<s:Dialog....>{contents}</s:Dialog>
So the contents should not be cleared out, so another configure was added that configures without clearing out the current contents of the container, which should remain set.